### PR TITLE
Sneak 100

### PR DESCRIPTION
**Pull request by liach**
Fixes for #1551

Signed-off-by: liach <liach@users.noreply.github.com>

### CODE REVIEW

File: mappings/net/minecraft/entity/Entity.mapping [hunk 1]
```diff
@@ -102,6 +102,14 @@ CLASS net/minecraft/class_1297 net/minecraft/entity/Entity
 		ARG 3 y
 		ARG 5 z
 	METHOD method_18276 isInSneakingPose ()Z
+		COMMENT Returns if the entity is in a crouching pose.
```
Comment (dblsaiko) [inline mappings/net/minecraft/entity/Entity.mapping]: if -> whether

File: mappings/net/minecraft/entity/Entity.mapping [hunk 2]
```diff
@@ -102,6 +102,14 @@ CLASS net/minecraft/class_1297 net/minecraft/entity/Entity
...
+		COMMENT
+		COMMENT <p>Compared to {@link #isSneaking()}, it only make the entity appear
```
Comment (dblsaiko) [inline mappings/net/minecraft/entity/Entity.mapping]: typo: makes

Comment (liach) [inline mappings/net/minecraft/entity/Entity.mapping]:
```suggestion
		COMMENT Returns whether the entity is in a crouching pose.
		COMMENT
		COMMENT <p>Compared to {@link #isSneaking()}, it only makes the entity appear
```

**Review by dblsaiko — APPROVED** [review]

**Review by YanisBft — APPROVED** [review]

**Review by i509VCB — APPROVED** [review]
**Illusion 100**

**Pull request merged**